DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.WebAPI/Controllers/ActivityController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Activity.External.WebAPI.Services;
+using GitHubExtension.Security.WebApi.Library.Exceptions;
+using Microsoft.AspNet.Identity;
+using Newtonsoft.Json.Linq;
+
+namespace GitHubExtension.Activity.External.WebAPI.Controllers
+{
+    [Authorize]
+    public class ActivityController : ApiController
+    {
+        public const string GetGitHubActivityRoute = "api/activity/external";
+        private IGitHubService _gitHubService;
+
+        public ActivityController(IGitHubService gitHubService)
+        {
+            _gitHubService = gitHubService;
+        }
+
+        [Route(GetGitHubActivityRoute)]
+        public async Task<IHttpActionResult> GetGitHubActivity([FromUri] int page)
```

Comment:
I want to create a page with GitHub events where you can navigate beetween pages with events with a help of pagination. This a number of page with github events. It will look like [that](https://developer.github.com/guides/traversing-with-pagination/#constructing-pagination-links)